### DISCUSSION TITLE
ジャンパの基盤を剥離させてしまったようでハンダがつかなくなってしまいました。

### DISCUSSION THREAD

**Discussion by Naoki-Egawa** [Yowkees/keyball#607] [General]
先日、keyball61を購入させていただきました。
頻出のトラブルで恐縮ですがジャンパのハンダ付を失敗してつかなくなってしまいました。

解決にどのような方法が取れるかご教示いただけないでしょうか。

右手トラックボール側です。

![image](https://github.com/Yowkees/keyball/assets/29029383/4ee7451d-1b10-4eab-8d9e-cd52f923111a)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

添付させていただいた写真の赤点どうしをエナメル線かゴム被覆線で繋いでいただければ復活できます。
こちらの作業は4Pピンソケット、タクトスイッチをはんだ付けした後に行ってください。

![OLEDランド剥がれ](https://github.com/Yowkees/keyball/assets/143698349/c4c8e473-c558-4ed9-bc49-0ad96f296b6c)

**Reply by Naoki-Egawa** [reply]
Ryo624 様

ご回答ありがとうございます。
試してみたところ、無事OLEDが正常に動作いたしました。
本当にありがとうございました。

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます。
ぜひ末永くご愛用いただけましたら幸いです。
